tests/MongoDB.Driver.Tests/MongoDatabaseImplTests.cs

```diff
@@ -554,6 +555,10 @@ public void Settings_should_be_set()
             [Values(false, true)] bool async)
         {
             RequireServer.Check();
+            if (usingSession)
+            {
+                RequireServer.Check().VersionGreaterThanOrEqualTo("3.6.0");
```

Comment:
We don't have a `Feature.Sessions` because whether sessions are supported or not depends on more than just the server version.